src/game/client/system/particlesystem/particlesys.cpp

```diff
@@ -65,6 +64,12 @@ ParticleSystem::ParticleSystem(const ParticleSystemTemplate *temp, ParticleSyste
     m_saveable(true),
     m_unkBool1(false)
 {
+    m_lastPos.Zero();
+    m_pos.Zero();
+    m_velCoefficient.Zero();
+    m_velCoefficient.x = 1.0;
```

Comment:
Use float literal 1.0f for float assignment.

Reply:
Fixed that.